config.json

```diff
@@ -3136,32 +3136,32 @@
     {
       "title": "General-purpose",
       "content": "Elixir is used for web development, embedded software, data ingestion, and multimedia processing.",
-      "icon": "TODO"
+      "icon": "general-purpose"
     },
     {
       "title": "Functional",
       "content": "Multi-clause functions with pattern matching and guards are the building blocks of Elixir code.",
-      "icon": "TODO"
+      "icon": "functional"
     },
     {
       "title": "Dynamically typed",
       "content": "Elixir has no compile-time type checks, favoring run-time pattern matching.",
-      "icon": "TODO"
+      "icon": "dynamically-typed"
     },
     {
       "title": "Immutable",
       "content": "All data in Elixir is immutable, allowing for safer and easier-to-reason-about concurrency.",
-      "icon": "TODO"
+      "icon": "easy"
```

Comment:
I think that one is a bit of a stretch, but I don't see any icon that would fit well.

Reply:
that's the icon: https://d31gpyts0tlbet.cloudfront.net/key-features/easy.svg does that fit?

Reply:
Alternatively:  https://d31gpyts0tlbet.cloudfront.net/key-features/homoiconic.svg